projects/zxing-scanner/src/lib/zxing-scanner.component.ts

```diff
@@ -81,6 +81,12 @@ export class ZXingScannerComponent implements AfterViewInit, OnDestroy {
   @Input()
   autofocusEnabled: boolean;
 
+  /**
+   * Modify delay between scans (default is 500ms)
+   */
+  @Input()
+  timeBetweenScansMillis = 500;
```

Comment:
I would just like to call it `timeBetweenScans` without the `Millis`.
```suggestion
  timeBetweenScans = 500;
```